src/features/projects/hooks/useProjects.ts

```diff
@@ -21,8 +21,25 @@ export function useProjectsById(ids: string[]) {
 const seed = 0;
 export function useSearchProjects(filterOverride?: Partial<Filter>) {
   const { data: filter } = useFilter("projects");
+  const clientFilter = filter;
+  if (filter) {
+    Object.keys(filter).forEach((k) => {
+      if (k === "seed" && typeof filter[k] === "string")
+        clientFilter[k] = parseInt(filter[k]);
+      else if (k === "cursor" && typeof filter[k] === "string")
+        clientFilter[k] = parseInt(filter[k]);
+      else if (k === "limit" && typeof filter[k] === "string")
+        clientFilter[k] = parseInt(filter[k]);
```

Comment:
You can use zod.number for these fields instead of conditions

Reply:
yep right, so this is more like a pseodo code concept here, after trying several conditions.

Reply:
I think it will be possible to do it with zod schemas and strip all the "null" | "undefined" values.